gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorage.java

```
@@ -421,6 +421,13 @@ void compose(String bucketName, List<String> sources, String destination, String
    */
   public AtomicLong getObjectStatistics(GoogleCloudStorageStatistics key);
 
+  /**
+   * To get the value object related statistics from GCS
+   *
+   * @param key name of the http related statistics key
+   */
+  public AtomicLong getHttpStatistics(GoogleCloudStorageStatistics key);
```

Let's add a default impl to throw unsupported Op exception